app/_layouts/docs.html

```diff
@@ -11,8 +11,8 @@
             <a target="_blank" href="https://github.com/Kong/docs.konghq.com/tree/master/app/{{page.path}}"><i class="fa fa-github"></i> Edit this Page</a>
           </div>
       <nav class="docs-navigation">
-        <a href="/" {% if page.edition == 'community' %} class="active"{% endif %}>Kong Community Edition (CE)</a>
-        <a href="/enterprise" {% if page.edition == 'enterprise' %} class="active"{% endif %}>Kong Enterprise Edition (EE)</a>
+        <a href="/" {% if page.edition == 'community' %} class="active"{% endif %}>Kong (CE)</a>
+        <a href="/enterprise" {% if page.edition == 'enterprise' %} class="active"{% endif %}>Kong Enterprise (EE)</a>
```

Comment:
Still says EE

Be careful with changes here - please be sure to test locally to ensure this doesn't break anything.